README.md

```diff
@@ -24,6 +24,8 @@ Instead, I propose that developers cryptographically sign the resources they int
 
 The prototype consists of two signed JavaScript files. One has been modified after it was signed. Only the signed file should execute in the browser. The prototype uses [OpenPGPJs](http://openpgpjs.org), and requires a modern browser. Just view tests/index.html and view the browser console.
 
+**Please note:** In practice, embedding the public key anywhere on the site where the content is also hosted (like in this example code) would be effectively useless. To help mitigate risk, it should instead be kept in a centralized and publicly accessible key store, or preferably, using a public key already retained or cached on the client's computer. That centralized mechanism isn't described here. This is only a *shim* which can be implemented in-browser in order to prevent further tampering by a potential hack on the server-side or MITM attack for plain text connections.
+
```

Comment:
I think lines are crossing, actually. While everything said above is true, what I was talking about in my tweet was the *private key* only: not the public key. My point was that the signing needs to be done separately from the subresource host, since the whole point is to detect a compromise of the subresource host. While perhaps obvious in the broad scheme, this point was significant to me because my thoughts originally went the direction of having the subresource host automatically sign the files. As @kojiromike clarified in response, that's not the intent: https://twitter.com/michaelasmith/status/676520007879143424